pkgs/development/python-modules/nbval/default.nix

```diff
@@ -0,0 +1,40 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, coverage
+, ipykernel
+, jupyter_client
+, nbformat
+, pytest
+, six
+, glibcLocales
+}:
+
+buildPythonPackage rec {
+  pname = "nbval";
+  version = "0.9.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "3f18b87af4e94ccd073263dd58cd3eebabe9f5e4d6ab535b39d3af64811c7eda";
+  };
+
+  LC_ALL = "en_US.UTF-8";
+
+  buildInputs = [ glibcLocales ];
+  propagatedBuildInputs = [ coverage ipykernel jupyter_client nbformat pytest six ];
+
+  # tests freeze
+  doCheck = false;
```

Comment:
You've disabled the check phase and you've done something for it below. Is this for like if the tests don't freeze anymore?